src/Strategy/Conda/CondaList.hs

```diff
@@ -0,0 +1,62 @@
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Strategy.Conda.CondaList (
+  analyze',
+  buildGraph,
+  CondaListDep(..)
+) where
+
+import Effect.Exec
+import qualified Data.Map.Strict as M
+import Control.Carrier.Diagnostics hiding (fromMaybe)
+import Graphing (Graphing, fromList)
+import Data.Aeson
+import Data.Text (Text)
+import Path
+import Types
+
+-- conda list --json will output dependency data in JSON format
+condaListCmd :: Command
+condaListCmd = Command
+  { cmdName = "conda"
+  , cmdArgs = ["list", "--json"]
+  , cmdAllowErr = Never
+  }
+
+buildGraph :: [CondaListDep] -> Graphing Dependency
+buildGraph deps = Graphing.fromList (map toDependency deps)
+  where
+    toDependency :: CondaListDep -> Dependency
+    toDependency CondaListDep{..} =
+      Dependency
+      {
+        dependencyType = CondaType,
+        dependencyName = listName,
+        dependencyVersion = CEq <$> listVersion,
+        dependencyLocations = [],
+        dependencyEnvironments = [],
+        dependencyTags = M.empty
+      }
+
+analyze' :: 
```

Comment:
This should just be `analyze`.  We only ever use `'` to denote multiple versions of the same function under different constraints or semantics.  Since there's no `analyze`, there should be no `analyze'`.

Btw, we may have this in other strategies, please point them out if you see them, those are (in my opinion) incorrect, and leftover from a refactor.